meilisearch/src/error.rs

```diff
@@ -95,6 +95,8 @@ pub enum PayloadError {
     MalformedPayload(serde_json::error::Error),
     #[error("A json payload is missing.")]
     MissingPayload,
+    #[error("Error while writing the playload to disk: `{0}`.")]
+    ReceivePayloadErr(Box<dyn std::error::Error + Send + Sync + 'static>),
```

Comment:
We need to rename this variant. There already is a `PayloadErr` part in the enum name.